src/App.tsx

```diff
@@ -29,128 +67,184 @@ export const App: React.FC = () => {
+                      value="Todo is being edited now"
+                    />
+                  </form>
+
+                  <div className="modal overlay">
+                    <div
+                      className="modal-background has-background-white-ter"
+                    />
+                    <div className="loader" />
+                  </div>
+                </div>
+
+                This todo is in loadind state
+                <div className="todo">
+                  <label className="todo__status-label">
+                    <input type="checkbox" className="todo__status" />
+                  </label>
+
+                  <span className="todo__title">Todo is being saved now</span>
+                  <button type="button" className="todo__remove">×</button>
+
+                  is-active class puts this modal on top of the todo
+                  <div className="modal overlay is-active">
+                    <div
+                      className="modal-background has-background-white-ter"
+                    />
+                    <div className="loader" />
+                  </div>
+                </div>
+                */}
         </section>
 
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer">
-          <span className="todo-count">
-            3 items left
-          </span>
-
-          {/* Active filter should have a 'selected' class */}
-          <nav className="filter">
-            <a href="#/" className="filter__link selected">
-              All
-            </a>
-
-            <a href="#/active" className="filter__link">
-              Active
-            </a>
-
-            <a href="#/completed" className="filter__link">
-              Completed
-            </a>
-          </nav>
-
-          {/* don't show this button if there are no completed todos */}
-          <button type="button" className="todoapp__clear-completed">
-            Clear completed
-          </button>
-        </footer>
+        {todos.length > 0 && (
+          <footer className="todoapp__footer">
+            <span className="todo-count">
+              {`${todos.length} items left`}
+            </span>
+
+            <nav className="filter">
+              <a
+                href="#/"
+                className={classNames('filter__link',
+                  { selected: sort === 'All' })}
+                onClick={() => setSort('All')}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={classNames('filter__link',
+                  { selected: sort === 'Active' })}
+                onClick={() => setSort('Active')}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={classNames('filter__link',
+                  { selected: sort === 'Completed' })}
```

Comment:
Literals like `Completed`, `Active`, `All` and so on  better to keep in some constants or enums. This bring you less potential typos and quick changes in case of this names updatings